paket.dependencies

```diff
@@ -1,15 +1,14 @@
 source https://www.nuget.org/api/v2/
 generate_load_scripts: on
 storage: none
-github fsprojects/FSharp.TypeProviders.SDK:28a24a69ada68ebd1ad25226634f4608e4875493 src/ProvidedTypes.fs
-github fsprojects/FSharp.TypeProviders.SDK:28a24a69ada68ebd1ad25226634f4608e4875493 src/ProvidedTypes.fsi
-github fsprojects/FSharp.TypeProviders.SDK:28a24a69ada68ebd1ad25226634f4608e4875493 src/ProvidedTypesTesting.fs
+github fsprojects/FSharp.TypeProviders.SDK src/ProvidedTypes.fsi
```

Comment:
I think it is better to keep the explicit commit here as well, less chance to mess things with paket update, because TP SDK breakage will only occur when changing the commit explicitly.

Reply:
The TypeProvider.SDK has been very stable recently. Even too stable now that Don seems to have other priorities. Thus I don't think that to be a big issue...

Reply:
Fixed.